modules/course-service/impl/src/main/java/com/innowise/educationalsystem/course/service/CourseService.java

```diff
@@ -0,0 +1,37 @@
+package com.innowise.educationalsystem.course.service;
+import com.innowise.educationalsystem.course.entity.Course;
+import com.innowise.educationalsystem.course.repository.CourseRepository;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.List;
+
+@Service
+public class CourseService {
```

Comment:
Create a generic crud service interface with delegation of the basic methods (like findAll, findById, etc.) to the corresponding repository. Contact me for details if isn't clear